src/Controls/src/Core/Page.cs

```diff
@@ -287,7 +287,9 @@ protected virtual bool OnBackButtonPressed()
 			var canceled = false;
 			EventHandler handler = (sender, args) => { canceled = true; };
 			window.PopCanceled += handler;
-			Navigation.PopModalAsync().ContinueWith(t => { throw t.Exception; }, CancellationToken.None, TaskContinuationOptions.OnlyOnFaulted, TaskScheduler.FromCurrentSynchronizationContext());
+			Navigation
```

Comment:
The other way just swallows this exception and leaves you confused why nothing is happening